src/eventProcessor.ts

```diff
@@ -216,13 +216,58 @@ export default async (event: ContractEvent): Promise<void> => {
       return;
     }
 
+    case ContractEventsSignatures.ColonyVersionAdded: {
+      const { version } = event.args;
+      const convertedVersion = BigNumber.from(version).toNumber();
+
+      verbose('New colony version:', convertedVersion, 'added to network');
+
+      await mutate('setCurrentVersion', {
+        input: {
+          key: 'colony',
```

Comment:
Only suggestion would be moving this to a constant e.g COLONY_VERSION_KEY and referencing `key: COLONY_VERSION_KEY`